core/store.go

```diff
@@ -183,6 +183,15 @@ func Keys(p string) ([]string, error) {
 	return keys, err
 }
 
+// GetDbSize returns all the keys present in the db
```

Comment:
we can change the comment to this
// GetDbSize returns the number of keys present in the db